PROJECT

```diff
@@ -86,47 +23,21 @@ resources:
     namespaced: true
   controller: true
   domain: bpfman.io
-  kind: BpfNsProgram
-  path: github.com/bpfman/bpfman-operator/apis/v1alpha1
-  version: v1alpha1
-- api:
-    crdVersion: v1
-    namespaced: true
-  controller: true
-  domain: bpfman.io
-  kind: XdpNsProgram
-  path: github.com/bpfman/bpfman-operator/apis/v1alpha1
-  version: v1alpha1
-- api:
-    crdVersion: v1
-    namespaced: true
-  controller: true
-  domain: bpfman.io
-  kind: TcNsProgram
-  path: github.com/bpfman/bpfman-operator/apis/v1alpha1
-  version: v1alpha1
-- api:
-    crdVersion: v1
-    namespaced: true
-  controller: true
-  domain: bpfman.io
-  kind: TcxNsProgram
+  kind: BpfNsApplication
```

Comment:
We agreed this would be `BpfApplication` and `ClusterBpfApplication`
But IIRC we wanted to get Billy's change in first before we changed the naming since this API change is the breaking one.

Reply:
Done